src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/ValidationMessages.java

```diff
@@ -23,7 +23,8 @@ private ValidationMessages() {}
     public static final String SECOND_NATIONALITY_SHOULD_BE_DIFFERENT = "%s should not be the same as the nationality given";
     public static final String DUPLICATE_TRUST_ID = "Duplicate Trust Id for %s";
     public static final String CEASED_DATE_BEFORE_START_DATE_ERROR_MESSAGE = "%s must be on or after the appointed date";
-    public static final String CEASED_DATE_BEFORE_CREATION_DATE_ERROR_MESSAGE = "%s must be on or after the creation date";
+    public static final String DATE_BEFORE_CREATION_DATE_ERROR_MESSAGE = "%s must be on or after the creation date";
```

Comment:
Trying to work out why this constant name had to change. Could you please explain.

Reply:
Wanted make this more generic as we are now testing the interested person date as well as the ceased date against the creation date. This should be checked as well I believe.

Reply:
I think we're testing the 'ceased date' against the creation date AND ALSO the date they became interested. This message seems to still be specifically used for testing the 'ceased date' against the creation date so I think it should be left as is. The new message you've defined - `CEASED_DATE_BEFORE_DATE_BECAME_INTERESTED_ERROR_MESSAGE` - is used for the second check.